pkg/agent/util/ipset/ipset.go

```diff
@@ -52,6 +54,14 @@ func NewClient() *Client {
 	return &Client{}
 }
 
+func (c *Client) DestroyIPSet(name string) error {
+	cmd := exec.Command("ipset", "destroy", name)
+	if err := cmd.Run(); err != nil {
```

Comment:
The method should be idempotent like others. If the ipset doesn't exist, it should not return error.

Reply:
According ipset' help: 

```-!
        Ignore errors when creating or adding sets or
        elements that do exist or when deleting elements
        that don't exist.
```

There is no option to ignore the error for ipset. Do you think it is okay that just ignore the error in the code or we read the error string in the code to verify that whether  the ipset exists?

Reply:
NotExist and Deletion failure should be differentiated.  Only the former should be ignored, otherwise we may end up with keep deleting a non-existing ipset forever like previous bugs. I assume at least the error string should have such information.

Reply:
Added some code to ignore the error of deleting a non-existing ipset.